plugins/node/opentelemetry-plugin-dns/src/utils.ts

```diff
@@ -105,25 +104,18 @@ const parseErrorCode = (code: string | undefined): Status => {
       case dns.NONAME:
       case dns.LOADIPHLPAPI:
       case dns.ADDRGETNETWORKPARAMS:
-        return { code: CanonicalCode.INTERNAL };
       case dns.SERVFAIL:
       case dns.NOTINITIALIZED:
       case dns.CONNREFUSED:
-        return { code: CanonicalCode.UNAVAILABLE };
       case dns.NOTFOUND:
-        return { code: CanonicalCode.NOT_FOUND };
       case dns.NOTIMP:
-        return { code: CanonicalCode.UNIMPLEMENTED };
       case dns.REFUSED:
-        return { code: CanonicalCode.RESOURCE_EXHAUSTED };
       case dns.CANCELLED:
-        return { code: CanonicalCode.CANCELLED };
       case dns.TIMEOUT:
-        return { code: CanonicalCode.DEADLINE_EXCEEDED };
       case dns.EOF:
-        return { code: CanonicalCode.OUT_OF_RANGE };
+        return { code: StatusCode.ERROR };
       default:
-        return { code: CanonicalCode.UNKNOWN };
+        return { code: StatusCode.UNSET };
```

Comment:
Shouldn't this be `StatusCode.ERROR` as well? previous implementation returned `CanonicalCode.UNKNOWN` in this case which is mapped to ERROR, not UNSET